graphene_django/rest_framework/mutation.py

```diff
@@ -67,9 +90,35 @@ def __init_subclass_with_meta__(cls, serializer_class=None,
         )
         super(SerializerMutation, cls).__init_subclass_with_meta__(_meta=_meta, input_fields=input_fields, **options)
 
+    @classmethod
+    def resolve_serializer_inputs(cls, root, info, **input):
+        lookup_field = cls._meta.lookup_field
+        model_class = cls._meta.model_class
+
+        if model_class:
+            if 'update' in cls._meta.model_operations and lookup_field in input:
+                instance = get_object_or_404(model_class, **{
+                    lookup_field: input[lookup_field]})
+            elif 'add' in cls._meta.model_operations:
+                instance = None
+            else:
+                raise Exception(
+                    'Invalid update operation. Input parameter "{}" required.'.format(
+                        lookup_field
+                    ))
+
+            return {
+                'instance': instance,
+                'data': input,
+                'partial': cls._meta.partial
```

Comment:
`instance` and `data` definitely make sense here, as those are very dynamic and dependent on the query. 

However, I think kwargs like `partial` (`many`, `context`, etc.) should use a different mechanism instead of living on `SerializerMutationOptions`. If we go with renaming this `get_serializer_kwargs`, those would be easy enough to add with a call to `super` on an overridden version.

There is also the compromise of adding a `serializer_kwargs` option to the `SerializerMutationOptions` that would be combined with this dict.

Reply:
Sounds good, I wasn't too happy with `partial` there